hail/python/test/hailtop/utils/test_gcs_requester_pays.py

```diff
@@ -10,7 +10,6 @@ def test_get_fs_by_requester_pays_config(cls):
     config_1 = "foo"
     config_2 = ("foo", ["bar", "baz", "bat"])
     kwargs_1 = {"gcs_requester_pays_configuration": config_1}
-    kwargs_2 = {"gcs_requester_pays_configuration": config_2}
```

Comment:
This is still a useful config test case, let's add an assert for it.

Reply:
lines 16-19 test that `fs[config_2]._gcs_kwargs` correctly match up with the individual members of `config_2`; is there something else to test with `kwargs_2` here that i'm missing?

Reply:
nope! I just missed those, thanks for pointing that out.